middleware-jackson/src/main/java/com/nytimes/android/external/store/middleware/jackson/JacksonStringParser.java

```diff
@@ -29,10 +30,10 @@ public JacksonStringParser(@Nonnull ObjectMapper objectMapper, @Nonnull Type typ
     }
 
     @Override
-    @Nullable
-    public Parsed call(@Nonnull String source) {
+    @SuppressWarnings({"PMD.EmptyCatchBlock"})
```

Comment:
Absolutely. What about adding a `throws Exception` to the parsers method signature and handling the errors from the caller?